### PR TITLE
mandrill api

### PR DESCRIPTION
**Pull request by pocesar**
This is a WIP, mainly to have the documentation through the typing, no params are documented / interfaced yet

### CODE REVIEW

File: mandrill-api/mandrill-api.d.ts [hunk 1]
```diff
@@ -0,0 +1,1182 @@
+// Type definitions for Mandrill API 1.x
+// Project: http://madrill.com
```
Comment (vvakame) [inline mandrill-api/mandrill-api.d.ts]: is this really? http://madrill.com/
https://mandrillapp.com/api/docs/ ?

Comment (pocesar) [inline mandrill-api/mandrill-api.d.ts]: oops, my bad

Comment (vvakame) [inline mandrill-api/mandrill-api.d.ts]: diff is too large...
What was changed?

Comment (pocesar) [inline mandrill-api/mandrill-api.d.ts]: only the URl at the beginning, sublime went full retard and changed all lines

**Comment by pocesar** [comment]
anything else wrong that is blocking this merge?

File: mandrill-api/mandrill-api.d.ts [hunk 2]
```diff
@@ -0,0 +1,1182 @@
+// Type definitions for Mandrill API 1.x
+// Project: http://mandrill.com/
```
Comment (vvakame) [inline mandrill-api/mandrill-api.d.ts]: this line point for http://madrill.com/ still. is it okey?

Comment (pocesar) [inline mandrill-api/mandrill-api.d.ts]: it was pointing to madrill.com (was missing the n). yes, it's mandrill from mailchimp

File: mandrill-api/mandrill-api.d.ts (hunk 1 above)
Comment (vvakame) [inline mandrill-api/mandrill-api.d.ts]: okey!

**Comment by vvakame** [comment]
@pocesar thanks! :+1:

**Pull request merged**